### ISSUE TITLE
Feature Request: Allow Tabs for Multiple Open Requests

### ISSUE THREAD

**Issue by joeltgray** [EsperoTech/yaade#64]
Something that would add a lot of functionality and improve the users work flow on Yaade would be adding tabs allowing for multiple open requests at once. This is something that Postman does brilliantly and even the likes of Hoppscotch have recently implemented. This would add a lot of value to Yaade and allow it to start competing directly with these other programs.

Seen in Hoppscotch here:
![image](https://github.com/EsperoTech/yaade/assets/15971045/f7d6793d-2b52-4244-a75d-dc6bce36a214)

**Comment by jonrosner** [comment]
Good idea. We could either have one tab per collection or one tab per request. What would you prefer?

**Comment by joeltgray** [comment]
I think one tab per open request would be ideal. It's what Hoppscotch and Postman do, so easy for people to change platform as the workflow is similar. Plus it could mean multiple instances/tabs of the same request could be open if you need to test two similar requests but don't necessarily want to save the request details.

How do you imagine a tab per collection would work? Maybe that is a better idea but I'm unsure how you intent the workflow to look?

Thanks for getting back to me by the way.
